website/docs/core-concepts/components/remote-state.md

```diff
@@ -0,0 +1,337 @@
+configure [VPC Flow Logs](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html) and store them in the S3 bucket.
+
+We will provision the two Terraform components in the `ue2-dev` Atmos stack (in the `dev` AWS account by setting `stage = "dev"` and in
+the `us-east-2` region by setting `environment = "ue2"`).
+
+<br/>
+
+## Configure and Provision `vpc-flow-logs-bucket` Component
+
+In the `stacks/catalog/vpc-flow-logs-bucket.yaml` file, add the following default configuration for the `vpc-flow-logs-bucket/defaults` Atmos
+component:
+
+```yaml title="stacks/catalog/vpc-flow-logs-bucket.yaml"
+components:
+  terraform:
+    vpc-flow-logs-bucket/defaults:
+      metadata:
+        # `metadata.type: abstract` makes the component `abstract`,
+        # explicitly prohibiting the component from being deployed.
+        # `atmos terraform apply` will fail with an error.
+        # If `metadata.type` attribute is not specified, it defaults to `real`.
+        # `real` components can be provisioned by `atmos` and CI/CD like Spacelift and Atlantis.
+        type: abstract
+      # Default variables, which will be inherited and can be overridden in the derived components
+      vars:
+        force_destroy: false
+        lifecycle_rule_enabled: false
+        traffic_type: "ALL"
+```
+
+<br/>
+
+In the `stacks/ue2-dev.yaml` stack config file, add the following config for the `vpc-flow-logs-bucket-1` Atmos component in the `ue2-dev` Atmos
+stack:
+
+```yaml title="stacks/ue2-dev.yaml"
+# Import the base Atmos component configuration from the `catalog`.
+# `import` supports POSIX-style Globs for file names/paths (double-star `**` is supported).
+# File extensions are optional (if not specified, `.yaml` is used by default).
+import:
+  - catalog/vpc-flow-logs-bucket
+
+components:
+  terraform:
+    vpc-flow-logs-bucket-1:
+      metadata:
+        # Point to the Terraform component in `components/terraform` folder
+        component: infra/vpc-flow-logs-bucket
+        inherits:
+          # Inherit all settings and variables from the 
+          # `vpc-flow-logs-bucket/defaults` base Atmos component
+          - vpc-flow-logs-bucket/defaults
+      vars:
+        # Define variables that are specific for this component
+        # and are not set in the base component
+        name: vpc-flow-logs-bucket-1
+        # Override the default variables from the base component
+        traffic_type: "REJECT"
+```
+
+<br/>
+
+Having the stacks configured as shown above, we can now provision the `vpc-flow-logs-bucket-1` Atmos component into the `ue2-dev` stack by executing
+the following Atmos commands:
+
+```shell
+atmos terraform plan vpc-flow-logs-bucket-1 -s ue2-dev
+atmos terraform apply vpc-flow-logs-bucket-1 -s ue2-dev
+```
+
+<br/>
+
+## Configure and Provision `vpc` Component
+
+Having the `vpc-flow-logs-bucket` Terraform component provisioned into the `ue2-dev` stack, we can now configure the `vpc` Terraform component
```

Comment:
I see now, so you're using "Terraform component" to mean literally the location on disk. While the "Atmos component" is the configuration of that component. The configuration could really be any type component.